libs/azure-storage/tests/utils.py

```diff
@@ -84,3 +99,52 @@ def get_test_blobs(
         return [blob for blob in _TEST_BLOBS if blob["blob_name"].startswith(prefix)]
 
     return _TEST_BLOBS
+
+
+def get_datalake_test_blobs(
+    blob_names: Optional[Union[str, Iterable[str]]] = None, prefix: Optional[str] = None
+) -> list[dict[str, Any]]:
+    if blob_names is not None:
+        if isinstance(blob_names, str):
+            blob_names = [blob_names]
+        updated_list = []
+        for name in blob_names:
+            for blob in _TEST_DATALAKE_BLOBS:
+                if blob["blob_name"] == name:
+                    updated_list.append(
+                        {
+                            **blob,
+                            "size": len(blob["blob_content"]),
+                            "metadata": blob.get("metadata", {}),
```

Comment:
For any of the metadata, if it is missing let's set the value to `None` instead of empty dictionary as it seems like when testing against the blob service the end value from the SDK is `None` if there is no metadata.